webapidemo/Controllers/BookController.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System.Collections.Generic;
+using Microsoft.AspNetCore.Mvc;
+
+namespace webapidemo.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class BookController : ControllerBase
+    {
+        BookService bookService;
+
+        public BookController()
+        {
+            bookService = new BookService();
+        }
+
+        // GET: api/Book
+        [HttpGet]
+        public List<Book> Get()
```

Comment:
You can implement the Custom Response object with {Data list and an error list} and return the result wrapped in that response object.